server/bot/poster.go

```diff
@@ -52,19 +52,19 @@ func (b *Bot) PostCustomMessageWithAttachments(channelID, customType string, att
 	return post, nil
 }
 
-// DM posts a simple Direct Message to the specified user
-func (b *Bot) DM(userID, format string, args ...interface{}) error {
-	return b.dm(userID, &model.Post{
-		Message: fmt.Sprintf(format, args...),
-	})
-}
-
-// DMWithAttachments posts a Direct Message that contains Slack attachments.
-// Often used to include post actions.
-func (b *Bot) DMWithAttachments(userID string, attachments ...*model.SlackAttachment) error {
-	post := model.Post{}
-	model.ParseSlackAttachment(&post, attachments)
-	return b.dm(userID, &post)
+// Post DM to the specified user
```

Comment:
Just to make this explicit enough:
```suggestion
// Post DM from the plugin bot to the specified user
```